http-gateway/service/getEvents_test.go

```diff
@@ -116,8 +116,7 @@ func TestRequestHandlerGetEvents(t *testing.T) {
 				accept:   uri.ApplicationProtoJsonContentType,
 				deviceID: "test",
 			},
-			wantErr:      true,
-			wantHTTPCode: http.StatusInternalServerError,
+			wantHTTPCode: http.StatusOK,
```

Comment:
Is this correct behavior? `StatusOK` when invalid deviceID is used?

Reply:
GetResources/GetDevices returns empty values when filter is not satisfied.

Reply:
It is similar as filter in gmail -> you don't get error but just empty body.